build/lin.sh

```diff
@@ -161,7 +161,7 @@ version_latest "lcms2" "$VERSION_LCMS2" "9815"
 version_latest "mozjpeg" "$VERSION_MOZJPEG" "mozilla/mozjpeg"
 version_latest "png" "$VERSION_PNG16" "1705"
 version_latest "spng" "$VERSION_SPNG" "24289"
-version_latest "webp" "$VERSION_WEBP" "webmproject/libwebp"
+#version_latest "webp" "$VERSION_WEBP" "webmproject/libwebp" # latest version on GitHub is a release candidate
```

Comment:
It looks like libwebp v1.4.0 has now been published (or at least tagged).

https://chromium.googlesource.com/webm/libwebp/+/refs/tags/v1.4.0

Reply:
Ah, our version sort placed release candidates (e.g. `v1.4.0-rc1`) _after_ the stable version (e.g. `v1.4.0`), so I assumed libwebp v1.4.0-rc1 was really the latest version.

Fixed by using `-c 'versionsort.suffix=-'` and `--sort='v:refname'` instead:
https://stackoverflow.com/a/12704727